modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaConnectionProviderModelParser.java

```diff
@@ -95,6 +97,7 @@ public JavaConnectionProviderModelParser(JavaExtensionModelParser extensionModel
         .orElse(ExtensionModel.class.getClassLoader());
 
     collectAdditionalModelProperties();
+    this.minMuleVersion = getMinMuleVersion().orElse(FIRST_MULE_VERSION);
```

Comment:
What would happen to this code later? My understanding is that this is added so that the test extensions actually calculate this, but in the final code this line would not exist.

Reply:
yes, this is not final code.

Reply:
Add a TODO pointing to a gus issue (you can use the one I am currently tackling)